radix-engine-interface/src/blueprints/resource/worktop.rs

```diff
@@ -1,17 +1,44 @@
 use crate::blueprints::resource::*;
+use crate::constants::RESOURCE_MANAGER_PACKAGE;
 use crate::data::scrypto::model::*;
 use crate::math::Decimal;
 use crate::*;
+use radix_engine_common::data::scrypto::*;
 use radix_engine_common::types::*;
 use sbor::rust::prelude::*;
+use sbor::*;
 
 pub const WORKTOP_BLUEPRINT: &str = "Worktop";
 
 pub const WORKTOP_DROP_IDENT: &str = "Worktop_drop";
 
 #[derive(Debug, Eq, PartialEq, ScryptoSbor)]
 pub struct WorktopDropInput {
-    pub worktop: Own,
+    pub worktop: OwnedWorktop,
+}
+
+#[derive(Debug, Eq, PartialEq, ScryptoCategorize, ScryptoEncode, ScryptoDecode)]
+#[sbor(transparent)]
+pub struct OwnedWorktop(pub Own);
+
+impl Describe<ScryptoCustomTypeKind> for OwnedWorktop {
+    const TYPE_ID: GlobalTypeId =
```

Comment:
We could make it well-known if you like! Would save some size in the ResourceManager schema, although don't know how much that matters.